docs/ui/auth/fragments/vue/overriding-sign-up-submit.md

```diff
@@ -0,0 +1,28 @@
+_App.vue_
+
+```html
+<template>
+	<div>
+        <amplify-authenticator>
+            <amplify-sign-up
+                header-text="My Custom Sign Up Text"
+                slot="sign-up"
+                :handleSignUp.prop="handleSignUp"
+            ></amplify-sign-up>
+        </amplify-authenticator>
+	</div>
+</template>
+<script>
+import { Auth } from '@aws-amplify/auth';
+
+export default {
+	name: 'Authenticator',
+	methods: {
+		handleSignUp(data) {
+			console.log(data);
```

Comment:
Why would want to `console.log()` this data?

Reply:
Nice catch, adjusting

Reply:
Found some other typos too so I pushed https://github.com/aws-amplify/docs/pull/3346/commits/70719cee6a891cc73b7f3332ce9732c620672d44 to handle those! The infamous @wlee221 clumsiness 😄